src/applications/claims-status/tests/components/claim-status-tab/RecentActivity.unit.spec.jsx

```diff
@@ -1034,13 +1040,15 @@ describe('<RecentActivity>', () => {
               getByText(
                 'Your claim moved into Step 3: Evidence gathering, review, and decision',
               );
-              getByText('We closed a request for "5103 Notice Response"');
+              getByText(
+                'We closed a request for "List of evidence we may need (5103 notice)"',
+              );
               expect($('va-pagination', container)).not.to.exist;
             });
           },
         );
         context(
-          'when cst5103UpdateEnabled and has an Automated 5103 Notice Response item',
+          'when cst5103UpdateEnabled disabled and has an Automated 5103 Notice Response item',
```

Comment:
Same as above; the below test has both feature flags disabled, so this needed to be updated to reflect that.